src/index.html

```diff
@@ -1,12 +1,388 @@
 <!doctype html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
+
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+
+      <div>
+        <a href="#home">
+          <img src="./images/logo.png" alt="moyo">
+        </a>
+      </div>
+
+      <nav class="nav">
+
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#apple" class="nav__link is-active">Apple</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#samsung" class="nav__link">Samsung</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#smartphones" class="nav__link">Smartphones</a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#laptops&computers"
+              class="nav__link"
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#gadgets" class="nav__link">Gadgets</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#tablets" class="nav__link">Tablets</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#photo" class="nav__link">Photo</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#video" class="nav__link">Video</a>
+          </li>
+        </ul>
+
+      </nav>
+
+    </header>
+    <main class="content">
+
+      <div class="card" data-qa="card">
+
+        <img
+          src="./images/imac.jpeg"
+          alt="imac"
+          class="card__logo"
+        >
+
+        <h2 class="card__name">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+
+        <p class="card__code">Product code: 195434</p>
+
+        <div class="card__rating">
+          <div class="stars">
+            <div class="stars__star stars__star--active"></div>
+            <div class="stars__star stars__star--active"></div>
+            <div class="stars__star stars__star--active"></div>
+            <div class="stars__star stars__star--active"></div>
+            <div class="stars__star"></div>
+          </div>
+          <div class="card__review">Reviews: 5</div>
+        </div>
+
+        <div class="card__price">
+          <span class="card__price-title">Price</span>
+          <span class="card__price-amount">$2,199</span>
+        </div>
+
+        <a
+          href="https://www.apple.com/ua/imac-24/"
+          data-qa="hover" class="card__buy-button"
+        >
+          Buy
+        </a>
+
+      </div>
+
+      <div class="card">
+
+        <img
+          src="./images/imac.jpeg"
+          alt="imac"
+          class="card__logo"
```

Comment:
It would be better another name
```suggestion
          class="card__image"
```